mkdocs.yml

```diff
@@ -295,7 +296,7 @@ plugins:
       integrations/transport/rsyslog.md: operation_center/data_collection/ingestion_methods/rsyslog.md
       integrations/transport/syslog-ng.md: operation_center/data_collection/ingestion_methods/syslog-ng.md
       integrations/sekoiaio_activity_logs.md: operation_center/integration_catalog/application/sekoiaio_activity_logs.md
-      integrations/thehive.md: operation_center/integration_catalog/application/thehive.md
+      integrations/imperva_waf.md: operation_center/integration_catalog/cloud_and_saas/imperva_waf.md
```

Comment:
It seems you overwrite the redirect for the hive.

Reply:
Fixed